### PR TITLE
Update main with lastest changes of develop

### PR DESCRIPTION
**Pull request by SwiftBeta**
Video de SwiftBeta explicando nuevas **funcionalidades** de Xcode 13

### CODE REVIEW

File: SwiftBetaXcode13/ContentView.swift [hunk 1]
```diff
@@ -9,8 +9,13 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        VStack {
```
Comment (SwiftBeta) [inline SwiftBetaXcode13/ContentView.swift:12]: Maybe here instead of using VStack you can use Form

File: SwiftBetaXcode13/ContentView.swift [hunk 2]
```diff
@@ -9,8 +9,13 @@ import SwiftUI
...
+            Text("Hello, world!")
+                .padding()
+            Button("Aceptar") {
```
Comment (SwiftBeta) [inline SwiftBetaXcode13/ContentView.swift:15]: Why do we need a Button here?

File: SwiftBetaXcode13/ContentView.swift [hunk 3]
```diff
@@ -9,8 +9,13 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        VStack {
+            Text("Hello, world!")
+                .padding()
```
Comment (LoopInfinitoBlog) [inline SwiftBetaXcode13/ContentView.swift:14]: Nice! maybe you can use 
```suggestion
                .padding(20)
```

File: SwiftBetaXcode13/ContentView.swift [hunk 4]
```diff
@@ -9,8 +9,13 @@ import SwiftUI
...
+            Button("Aceptar") {
+                print("Execute API...")
```
Comment (LoopInfinitoBlog) [inline SwiftBetaXcode13/ContentView.swift:16]: Maybe we can change this message

**Review by LoopInfinitoBlog — APPROVED** [review]

**Pull request merged**